src/test/java/uk/ac/cam/cl/dtg/isaac/api/IsaacIntegrationTest.java

```diff
@@ -136,6 +136,7 @@ public abstract class IsaacIntegrationTest {
   protected static RecaptchaManager recaptchaManager;
   protected static PgUsers pgUsers;
   protected static PgAnonymousUsers pgAnonymousUsers;
+  protected static ContentMapper contentMapper;
```

Comment:
Small one, but this and the two lines above are in a block of code called "Managers", so they probably belong somewhere else.

Reply:
Oh, possibly. I overlooked the header and just put it roughly in the order it was created. I'll have a look and figure out a better place for it...